syntax = "proto2";

package TORCS_Sensors;

option java_outer_classname = "Sensor_Message";

message Sensors {
	optional float angle = 1;
	optional float currentLabTime = 2;
	optional float damage = 3;
	optional float distanceFromStart = 4;
	optional float totalDistanceFromStart = 5;
	optional float distanceRaced = 6;
	optional float fuel = 7;
	optional float gear = 8;
	optional float lastLapTime = 9;
	optional float opponents = 10;
	optional float racePos = 11;
	optional float rpm = 12;
	optional float speedX = 13;
	optional float speedY = 14;
	optional float speedZ = 15;
	optional float track = 16;
	optional float trackPosition = 17;
	optional float wheelSpinVelocity = 18;
	optional float z = 19;
	optional float focus = 20;
}

